cmake/add_pstore.cmake

```diff
@@ -266,6 +266,47 @@ function (add_pstore_executable target_name)
 endfunction(add_pstore_executable)
 
 
+############################
+# add_pstore_install_targets
+############################
+
+function(add_pstore_install_targets target)
```

Comment:
Should this be add_pstore_install_target (singular) since it creates just a single target?

Reply:
Good point. Changing to singular name.